packages/babel-generator/src/node/index.ts

```diff
@@ -80,19 +79,15 @@ function isOrHasCallExpression(node: t.Node): boolean {
 export function needsWhitespace(
   node: t.Node,
   parent: t.Node,
-  type: "before" | "after",
+  type: number,
 ): boolean {
   if (!node) return false;
 
   if (isExpressionStatement(node)) {
     node = node.expression;
   }
 
-  let linesInfo: WhitespaceObject | null | boolean = find(
-    expandedWhitespaceNodes,
-    node,
-    parent,
-  );
+  let linesInfo: any = find(expandedWhitespaceNodes, node, parent);
```

Comment:
you should never use `any`

Reply:
This is actually because the enumeration bit operations across files cannot be constant. I would try not to use `any` though, maybe `number`?

Reply:
Ah, your review helped a lot! I found a bug that was not covered by the test!

Can anyone else help provide a relevant test? I'm not familiar with it here.

![image](https://user-images.githubusercontent.com/30521560/176570555-102fef94-8438-4b8d-9729-b4af65dd109d.png)